include/seqan3/contrib/span/span

```diff
@@ -0,0 +1,436 @@
+// -*- C++ -*-
```

Comment:
Do we need this? What does it mean, just that it is C++ code?

Reply:
Yes, this is an "editor" standard to set the file extension for extension less files.